R/write.R

```diff
@@ -12,7 +12,8 @@
 #' @param strict_checks If TRUE, xpt validation will report errors and not write
 #'   out the dataset. If FALSE, xpt validation will report warnings and continue
 #'   with writing out the dataset. Defaults to FALSE
-#' @inheritParams xportr_length
+#' @inheritParams xportr_df_label
+#' @inheritSection xportr_df_label Metadata
```

Comment:
`metadata` parameter should be the same as `xportr_df_label` (Metacore or with dataset name)